config/schema_descriptions.yml

```diff
@@ -32,23 +32,54 @@ reporting_facility_states:
+    adjusted_bs_over_300_under_care: "The number of diabetic patients assigned to the facility that were registered before the last 3 months, with blood sugar value categorized as bs_over_300. Dead and lost to follow-up patients are excluded."
+
     adjusted_missed_visit_lost_to_follow_up: "adjusted_missed_visit_lost_to_follow_up"
+    adjusted_bs_missed_visit_lost_to_follow_up: "adjusted_diabetes_missed_visit_lost_to_follow_up"
     adjusted_visited_no_bp_lost_to_follow_up: "adjusted_visited_no_bp_lost_to_follow_up"
     adjusted_patients_under_care: "The number of hypertensive patients assigned to the facility that were registered before the last 3 months"
+    adjusted_diabetes_patients_under_care: "The number of diabetic patients assigned to the facility that were registered before the last 3 months"
     adjusted_patients_lost_to_follow_up: "The number of hypertensive patients assigned to the facility that were registered before the last 3 months, with no visit in the last year"
+    adjusted_diabetes_patients_lost_to_follow_up: "The number of diabetic patients assigned to the facility that were registered before the last 3 months, with no visit in the last year"
     monthly_cohort_controlled: "The number of patients from `monthly_cohort_patients` with a BP <140/90 at their latest visit in the next two months. Eg. The number of patients registered in Jan 2020 with a controlled BP at their latest visit in Feb-Mar 2020"
     monthly_cohort_uncontrolled: "The number of patients from `monthly_cohort_patients` with a BP >=140/90 at their latest visit in the next two months. Eg. The number of patients registered in Jan 2020 with an uncontrolled BP at their latest visit in Feb-Mar 2020"
     monthly_cohort_missed_visit: "The number of patients from `monthly_cohort_patients` with no visit in the next two months. Eg. The number of patients registered in Jan 2020 with no visit in Feb-Mar 2020"
     monthly_cohort_visited_no_bp: "The number of patients from `monthly_cohort_patients` with no BP recorded at their latest visit in the next two months. Eg. The number of patients registered in Jan 2020 with no BP recorded at their latest visit in Feb-Mar 2020"
     monthly_cohort_patients: "The number of patients assigned to the facility that were registered two months before the reporting month, and have a . Eg. For a March 2020 report, the number of patients registered in Jan 2020"
     monthly_overdue_calls: "The number of overdue calls made by healthcare workers at the facility in the reporting month"
-    monthly_follow_ups: "The number of follow-up patient visits at the facility in the reporting month"
+    monthly_follow_ups: "The number of hypertensive follow-up patient visits at the facility in the reporting month"
```

Comment:
We've created a separate story to rename the hypertension columns: https://app.shortcut.com/simpledotorg/story/7950/clean-up

We debated renaming the columns in this PR, but it would result a large set of seemingly unrelated changes, and also all the metabase questions would need to be updated. Since this can be done once the DM reports are ready, we are choosing to them at the end of the epic.

Reply:
We've not removed `visited_no_bp_lost_to_follow_up` column in this PR for the same reason.

Reply:
Great plan 👍